### DISCUSSION TITLE
Mojo 🔥 Advent of Code 2023 Solutions

### DISCUSSION THREAD

**Discussion by jackos** [modular/modular#1329] [Show and tell]
Post a link to your GitHub repo with Mojo 🔥 [Advent of Code 2023](https://adventofcode.com/) solutions here.

Discuss solutions in our #advent-of-code channel on [Discord.](https://modul.ar/discord)

We regularly send Mojo swag to active community members, so get involved and you might have a box of goodies coming your way!

**Comment by pmatos** [comment]
Solutions soon in mojo🔥 in a repository near you: https://github.com/pmatos/aoc2023

**Reply by pmatos** [reply]
That was an interesting start. I found some obstacles but will discuss those on Discord.

**Comment by harskish** [comment]
Feel free to drop by and steal a utility function or two :)
https://github.com/harskish/aoc23

**Comment by rd4com** [comment]
Hello, here is the repo: https://github.com/rd4com/mojo_aoc_23/

**Comment by isuckatcs** [comment]
Repo with Mojo 🔥 solutions too: https://github.com/isuckatcs/advent-of-code

**Comment by edzhuang** [comment]
Looking forward to learning Mojo https://github.com/EdZ543/yet-another-aoc-solution-repo/tree/main/2023

**Comment by velaia** [comment]
A little late to the party but having fun and started using v 0.6.0 (d55c0025) 😉 string splitting functionality for day 2 
 https://github.com/velaia/adventofcode2023mojo

I've been tempted to import Python a couple of times (e.g. stripping of whitespaces) but managed not to so far 😀

**Comment by cyrillzadra** [comment]
🔥https://github.com/cyrillzadra/mojo_adventofcode 🔥 Now at Day 2 Part 2

**Comment by rizal-razuwan** [comment]
I try my best to finish it... 👨🏻‍💻🔥  https://github.com/rizal-razuwan/mojo-aoc23

**Comment by velaia** [comment]
Hey all Mojo advent of coders! 😃

How was everyone's experience participating?

I've made it to day 4 with Mojo but got quite frustrated by how slow I was and some of the inconvenient ways I had to resort to compared to my beloved Python. But I did like the coding challenges. So I continued till about day 16 with Python and suddenly appreciated all the "convenience features" of the language a lot more 😆

My personal résumé is that at the 0.6 version I wouldn't have exchanged Python for Mojo in my personal projects yet. But I do like the mission of Modular with Mojo and would love to see it succeed and be part of that path to success (somehow?). If I come upon more compute intense projects in my future work I will definitely consider Mojo again. And I'll set a reminder for 2024's AOC to take a look at the state of Mojo again 😃🔥
